nexus/src/db/queries/external_ip.rs

```diff
@@ -31,10 +31,9 @@ type FromClause<T> =
 type IpPoolRangeFromClause = FromClause<schema::ip_pool_range::table>;
 const IP_POOL_RANGE_FROM_CLAUSE: IpPoolRangeFromClause =
     IpPoolRangeFromClause::new();
-type InstanceExternalIpFromClause =
-    FromClause<schema::instance_external_ip::table>;
-const INSTANCE_EXTERNAL_IP_FROM_CLAUSE: InstanceExternalIpFromClause =
-    InstanceExternalIpFromClause::new();
+type ExternalIpFromClause = FromClause<schema::external_ip::table>;
+const INSTANCE_EXTERNAL_IP_FROM_CLAUSE: ExternalIpFromClause =
```

Comment:
Chop the `INSTANCE_` prefix from the const?

Reply:
Done!